### PR TITLE
Release notes for v2.11.18

### PR DESCRIPTION
**Pull request by qiuosier**

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
 Release Notes
 =============
 
+2.11.18
+-------
+Release date: September 19, 2024
+
+* Added ``with_artifact()`` in ``ContainerRuntime`` class to support running container job with additional artifact.
```
Comment (mrDzurb) [inline docs/source/release_notes.rst:9]: DId we update our docs for this feature?

**Review by mrDzurb — APPROVED** [review]

File: docs/source/release_notes.rst (hunk 1 above)
Comment (qiuosier) [reply docs/source/release_notes.rst:9]: Not yet :)

**Review by mrDzurb — APPROVED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -94,6 +94,7 @@ data = [
 ]
 geo = [
   "geopandas<1.0.0",  # in v1.0.0 removed the built-in dataset 'naturalearth_lowres', fix when relax version of geopandas needed
+  "fiona<=1.9.6",
```
Comment (mrDzurb) [inline pyproject.toml:97]: Third party licenses need to be updated as well?

**Review by mrDzurb — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (VipulMascarenhas) [reply pyproject.toml:97]: +1.

**Review by VipulMascarenhas — APPROVED** [review]

**Pull request merged**